### ISSUE TITLE
It seems not supoort enum

### ISSUE THREAD

**Issue by chank** [Distelli/graphql-apigen#26]
I defined the following schema:
```
enum VerificationEnum {
  PHONE
  EMAIL
}

type VerificationPayload {
  verificationType: VerificationEnum
}
```

I used it with spring, but it failed with the following Exception:
Caused by: graphql.GraphQLException. type VerificationEnum not found in schema

**Comment by brimworks** [comment]
Can you provide more information? Perhaps an easy way to reproduce the problem? enums work in our unit tests:

https://github.com/Distelli/graphql-apigen/blob/master/apigen/src/test/projects/starwars/schema/starwars.graphql

Thanks,
-Brian

**Comment by chank** [comment]
I use it with spring boot.

**Comment by chank** [comment]
This is a demo project.

https://bitbucket.org/chankflq/spring-graphql-apigen-demo

**Comment by gdelafosse** [comment]
Hello,
I reproduce too.
It seems that the generator doesn't create a TypeResolver for the Enum.
Your unit test only tests that the generation is successful and actually this error is visibile only at runtime.

**Comment by brimworks** [comment]
Please verify this works as expected:

https://github.com/Distelli/graphql-apigen/pull/29

...the unit test makes it appear that this is fixed.

**Comment by seetharamani** [comment]
@brimworks  I verified this fix and it works fine. Thank you!

**Comment by brimworks** [comment]
Released as version 4.0.0.
